src/grpc/server.rs

```diff
@@ -97,11 +98,10 @@ impl ConsumerService for TaskbrokerServer {
 
         if let Ok(Some(inflight_activation)) = update_result {
             let duration = Utc::now() - inflight_activation.added_at;
-            let activation = inflight_activation.activation;
             metrics::histogram!(
                 "task_execution.conclusion.duration",
-                "namespace" => activation.namespace.clone(),
-                "taskname" => activation.taskname.clone(),
+                "namespace" => inflight_activation.namespace.clone(),
+                "taskname" => inflight_activation.taskname.clone(),
```

Comment:
Do we still need to clone here?

Reply:
Yes, those values get used again further down.